src/wasm-linker.cpp

```diff
@@ -230,8 +230,9 @@ void Linker::layout() {
   }
 
   // finalize function table
-  if (out.wasm.table.segments.size() > 0) {
-    out.wasm.table.initial = out.wasm.table.max = getTableSegment().data.size();
+  unsigned int tableSize = getTableData().size();
```

Comment:
In my head, `getTableDataRef` is useful for when you want to modify the table data, and `getTableData` is for when you want to read it. Getting the ref directly will be faster, returning a copy is more in line with that intent.